docs/javascript/browser-apis-and-jquery.md

```diff
@@ -355,4 +355,52 @@ You can select multiple elements as well. For example, elements that all share t
 <figure markdown>
 ![jquery get elements example](https://github.com/codewizardshq/docs/blob/main/docs/assets/browser-apis-and-jquery/get-elements-by-class-jquery.png?raw=true){ width="100%" }
 <figcaption></figcaption>
-</figure>
\ No newline at end of file
+</figure>
+
+//what is needed?
```

Comment:
Let's leave comments out of the docs so that folks don't see them when they visit the live site.